RELEASE_NOTES.md

```diff
@@ -52,5 +52,20 @@ For a list of all features with an announced removal target, see [Removals](intr
 ## Known issues
 
 * UAIs use a default route that sends outbound packets over the CMN, this will be addressed in a future release so that the default route uses the CAN/CHN.
+* The Slurm installer released in CPE 23.03 (`cpe-slurm-23.03-sles15-1.2.10.tar.gz`) has an issue that causes failures when installed with the IUF. To work around the issue, run the following commands before the IUF `process-media` stage:
```

Comment:
This looks good, but perhaps we could add a note indicating what to do if they fail to do this prior to installation.  Something like.

"If Slurm 23.03 is installed without the workaround in place, apply the workaround and re-install it with the '--force' option"